moveit_ros/occupancy_map_monitor/include/moveit/occupancy_map_monitor/occupancy_map_monitor.h

```diff
@@ -56,11 +56,9 @@ namespace occupancy_map_monitor
 class OccupancyMapMonitor
 {
 public:
-  OccupancyMapMonitor(const std::shared_ptr<tf2_ros::Buffer>& tf_buffer, const std::string& map_frame = "",
-                      double map_resolution = 0.0);
-  OccupancyMapMonitor(double map_resolution = 0.0);
-  OccupancyMapMonitor(const std::shared_ptr<tf2_ros::Buffer>& tf_buffer, ros::NodeHandle& nh,
+  OccupancyMapMonitor(const std::shared_ptr<tf2_ros::Buffer>& tf_buffer, const rclcpp::Node::SharedPtr& node,
```

Comment:
For consistency `node` should be the first parameter

Reply:
Done as suggested.